### PR TITLE
Decode Sai CDP migration

### PR DESCRIPTION
**Pull request by prettyirrelevant**

### CODE REVIEW

File: rotkehlchen/chain/ethereum/modules/makerdao/sai/decoder.py [hunk 1]
```diff
@@ -513,6 +515,41 @@ def _decode_peth_mint_after_deposit(
 
         return None, []
 
+    def _decode_sai_cdp_migration(
+            self,
+            token: Optional[EvmToken],  # pylint: disable=unused-argument
+            tx_log: EvmTxReceiptLog,
+            transaction: EvmTransaction,  # pylint: disable=unused-argument
```
Comment (nebolax) [inline rotkehlchen/chain/ethereum/modules/makerdao/sai/decoder.py:522]: transaction is not unused here

**Review by nebolax — COMMENTED** [review]
Also is missing a test. Can you add one?

File: rotkehlchen/chain/ethereum/modules/makerdao/sai/decoder.py [hunk 2]
```diff
@@ -513,6 +514,41 @@ def _decode_peth_mint_after_deposit(
...
+
+        event = self.base.make_event_from_transaction(
+            transaction=transaction,
+            tx_log=tx_log,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_ETH,
+            balance=Balance(),
+            location_label=owner,
+            notes=f'Migrate Sai CPD {old_cdp_id} to Dai CDP {new_cdp_id}',
```
Comment (LefterisJP) [inline rotkehlchen/chain/ethereum/modules/makerdao/sai/decoder.py:542]:
```suggestion
            notes=f'Migrate Sai CDP {old_cdp_id} to Dai CDP {new_cdp_id}',
```

File: rotkehlchen/tests/unit/decoders/test_makerdao_sai.py [hunk 3]
```diff
@@ -2268,3 +2269,31 @@ def test_makerdao_sai_proxy_interaction(ethereum_transaction_decoder):
...
+    tx_hex = deserialize_evm_tx_hash('0x03620c6bf5edb7a7935953337ffcfac70d631cf2012d6c80d36828d636063318 ')  # noqa: E501
+    evmhash = deserialize_evm_tx_hash(tx_hex)
+    user_address = ethereum_accounts[0]
+    events, _ = get_decoded_events_of_transaction(
+        evm_inquirer=ethereum_transaction_decoder.evm_inquirer,
+        database=ethereum_transaction_decoder.database,
+        tx_hash=evmhash,
+    )
+    # check that the last event is the migration event, and it is decoded properly
+    assert events[-1] == HistoryBaseEntry(
```
Comment (LefterisJP) [inline rotkehlchen/tests/unit/decoders/test_makerdao_sai.py:2287]: Why not also confirm the other events?

File: docs/changelog.rst [hunk 4]
```diff
@@ -2,8 +2,9 @@
 Changelog
 =========
 
+* :feature:`-` Transactions involving Sai CDP migration to Dai CDP are now properly decoded.
 * :bug:`-` Fixed a bug where token balance detection for EVM tokens with many addresses may not have worked properly.
-* :bug:`-` Fixed a bug where an error message is not showen when some input in the UI is filled with an incorrect value.
+* :bug:`-` Fixes bug where error message is not showed when some input filled with incorrect value.
```
Comment (LefterisJP) [inline docs/changelog.rst:7]:
```suggestion
* :bug:`-` Fixes bug where error message is not shown when some input filled with incorrect value.
```

File: docs/changelog.rst [hunk 5]
```diff
@@ -2,8 +2,9 @@
 Changelog
 =========
 
+* :feature:`-` Transactions involving Sai CDP migration to Dai CDP are now properly decoded.
 * :bug:`-` Fixed a bug where token balance detection for EVM tokens with many addresses may not have worked properly.
-* :bug:`-` Fixed a bug where an error message is not showen when some input in the UI is filled with an incorrect value.
+* :bug:`-` Fixes bug where error message is not shown when some input filled with incorrect value.
```
Comment (LefterisJP) [inline docs/changelog.rst:7]:
```suggestion
* :bug:`-` Fixes bug where an error message is not shown when some input filled with incorrect value.
```

**Review by LefterisJP — APPROVED** [review]
LGTM. Spelling later

**Pull request merged**